lib/monitoring/aws-apigatewayv2/ApiGatewayV2HttpApiMonitoring.ts

```diff
@@ -98,10 +136,10 @@ export class ApiGatewayV2HttpApiMonitoring extends Monitoring {
     const namingStrategy = new MonitoringNamingStrategy({
       ...props,
       namedConstruct: props.api,
-      fallbackConstructName: props.api.httpApiId,
-      humanReadableName: `${props.api.httpApiId} ${
-        props.apiStage ?? "$default"
-      } ${props.apiMethod ?? ""} ${props.apiResource ?? ""}`,
+      fallbackConstructName: props.api.apiId,
```

Comment:
Is there a tangible difference?

Reply:
It should not be - they just deprecated the httpApiId and advised us to use the latter.